code/datums/brain_damage/mild.dm

```diff
@@ -141,7 +141,7 @@
 	else if(owner.get_active_held_item())
 		var/drop_chance = 1
 		var/obj/item/I = owner.get_active_held_item()
-		drop_chance += I.w_class
+		drop_chance += max(round(log(10, I.atom_size) - 2), 0)
```

Comment:
This seems infinitely more obnoxious to work with than the defines being 1 2 3 4